db/migrate/20210118145940_drop_pull_request_sizes_table.rb

```diff
@@ -0,0 +1,11 @@
+class DropPullRequestSizesTable < ActiveRecord::Migration[6.0]
+  def change
+    def up
+      drop_table :pull_request_sizes
+    end
+
+    def down
+      raise ActiveRecord::IrreversibleMigration
```

Comment:
Why do we want to make this migration irreversible? We could easily make it reversible, and I don't think having irreversible migrations is desirable.